osu.Game/Overlays/Profile/Sections/PaginatedContainer.cs

```diff
@@ -11,22 +11,27 @@
 using osu.Game.Online.API;
 using osu.Game.Rulesets;
 using osu.Game.Users;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
 
 namespace osu.Game.Overlays.Profile.Sections
 {
-    public abstract class PaginatedContainer : FillFlowContainer
+    public abstract class PaginatedContainer<T> : FillFlowContainer
```

Comment:
Ensure given argument is an `APIRequest`
```suggestion
    public abstract class PaginatedContainer<T> : FillFlowContainer
        where T : APIRequest
```

Reply:
Please read the code. T is not an api request. T is response of created request.

Reply:
oh sorry, nvm this then